### ISSUE TITLE
Update process for pis

### ISSUE THREAD

**Issue by smathermather** [OpenDroneMap/odm360#135]
We need to be able to trigger updates on the pis. This should probably include the following:
* Update cache on parent pi
* A way to trigger child pis to do their own updates.

**Comment by hcwinsemius** [comment]
I can add an update button on the parent pi that issues an 'update' task to all children, awaits if update process return success on all children and then updates itself. Does that make sense?

**Comment by smathermather** [comment]
Yes, that'd be perfect. Thinking of low bandwidth locations, we'll want to also set up an update cache, but if you implement that button, I can modify the provisioning scripts to handle caching of updates.

**Comment by hcwinsemius** [comment]
http://tuts.nanodesignsbd.com/en/releasing-github-project/
